### PR TITLE
Add processing function

### PR DESCRIPTION
**Pull request by KPostOffice**
Add processing function which I have been using for aggregation.

### CODE REVIEW

File: aggregators.py [hunk 1]
```diff
@@ -29,26 +33,36 @@
 HI_SEV_WEIGHT = 100
 
 
-def sample_aggregator(si_bandit_results: Dict[str, Any], si_cloc_results: Dict[str, Any], **kwargs) -> Dict[str, Any]:
+def sample_aggregator(si_bandit_results: Dict[str, Any], si_cloc_results: Dict[str, Any], **kwargs) -> float:
     """Run an example of an aggregator using results from si-cloc and si-bandit."""
+    print(si_cloc_results)
+    print(si_bandit_results)
```
Comment (harshad16) [inline aggregators.py]: is this intentional?

File: app.py [hunk 2]
```diff
@@ -100,15 +111,15 @@ def si_aggregator(
     if agg_func is None:
         raise NotImplementedError(f"{aggregation_func} aggregation function not implemented yet.")
 
+    print(os.listdir("/mnt/workdir"))
+
```
Comment (harshad16) [inline app.py]: is this intentional as well?

File: openshift/buildConfig-template.yaml [hunk 3]
```diff
@@ -0,0 +1,121 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
```
Comment (harshad16) [inline openshift/buildConfig-template.yaml]: this is moved to thoth-application

File: openshift/imageStream-template.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+apiVersion: template.openshift.io/v1
```
Comment (harshad16) [inline openshift/imageStream-template.yaml]: this is moved to thoth-appplication

**Review by harshad16 — COMMENTED** [review]
lgtm :+1: 
just some minor comments
also we have migrated the openshift manifest to thoth-application

**Review by harshad16 — COMMENTED** [review]
lgtm, please can you rebase and squash the commit to one.
thanks :100:

**Comment by KPostOffice** [comment]
See #14

**Pull request closed without merging**